package.json

```diff
@@ -26,6 +26,7 @@
         "webpack-dev-server": "^3.10.1"
     },
     "dependencies": {
+        "-": "0.0.1",
```

Comment:
What is this? "-" ??  Is it necessary for parsons?

Reply:
Oh, I didn't make any change in package.json.